src/index.html

```diff
@@ -9,6 +9,34 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo-link" href="#home">
+        <img
+          class="logo"
+          src="./images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a class="nav__link is-active" href="#">Apple</a></li>
```

Comment:
please add empty lines between the li blocks, check the checklist 
<img width="1260" alt="Screenshot 2023-10-05 at 13 16 30" src="https://github.com/mate-academy/layout_moyo-header/assets/64746943/409af7a5-aeff-44c5-9dfc-db8f61ccbd53">